settings.yml

```diff
@@ -14,6 +14,17 @@ hide_if_no_description:
   default: true
   description: "Hide banners if category description is not set"
 
+show_category_logo:
+  default: true
```

Comment:
Maybe this should be `default: false` so current implementations remain unchanged?